tensorflow_encrypted/layers/convolution.py

```diff
@@ -80,24 +80,25 @@ def backward(self, d_y, learning_rate):
             raise TypeError("channels must be first on the backward pass")
 
         x = self.cache
-        h_filter, w_filter, d_filter, n_filter = map(int, self.weights.shape)
+        h_filter, w_filter, d_filter, n_filter = self.fshape
 
-        if self.model.layers.index(self) != 0:
-            W_reshaped = self.weights.reshape(n_filter, -1).transpose()
-            dout_reshaped = d_y.transpose(1, 2, 3, 0).reshape(n_filter, -1)
-            dx = W_reshaped.dot(dout_reshaped).col2im(imshape=self.cached_input_shape,
-                                                      field_height=h_filter,
-                                                      field_width=w_filter,
-                                                      padding=self.padding,
-                                                      stride=self.strides)
+        dx = None
```

Comment:
is this on purpose, including the commented code below?

Reply:
yes it is on purpose, it was pseudocode for when I start with implementing the dx computation from the privateml repo (probably tomorrow). But it does not need to be in master.. Do you want me to remove it?

Reply:
no worries, just wanted to know! you'll keep working on this branch?

Reply:
yes